src/command.rs

```diff
@@ -26,27 +26,28 @@ use crate::{generator, generator::Language};
 use std::path::PathBuf;
 
 use async_recursion::async_recursion;
-use eyre::{ensure, Context, ContextCompat};
+use eyre::Context;
 use semver::{Version, VersionReq};
 use std::{env, path::Path, sync::Arc};
 
 const INITIAL_VERSION: Version = Version::new(0, 1, 0);
 
 /// Initializes the project
 pub async fn init(kind: PackageType, name: Option<PackageName>) -> eyre::Result<()> {
-    ensure!(
-        !Manifest::exists().await?,
-        "Cannot re-initialize an existing project"
-    );
-
-    const DIR_ERR: &str = "Failed to read current directory name";
+    if Manifest::exists()
+        .await
+        .wrap_err("failed to access filesystem")?
+    {
+        eyre::bail!("a manifest file was found, project is already initialized");
+    }
 
     fn curr_dir_name() -> eyre::Result<PackageName> {
-        std::env::current_dir()?
+        std::env::current_dir()
+            .wrap_err("failed to access current directory")?
             .file_name()
-            .wrap_err(DIR_ERR)?
+            .expect("unexpected error: current directory path terminates in ..")
```

Comment:
```suggestion
            .wrap_err("unexpected error: current directory path terminates in ..")?
```

Reply:
Can we do it for consistency? I know you explained to to me, your reasoning is valid, this is just a OCD nit